### DISCUSSION TITLE
A repository with docs

### DISCUSSION THREAD

**Discussion by alies-dev** [telegram-bot-sdk/telegram-bot-sdk#64] [Ideas]
I found https://github.com/telegram-bot-sdk/docs/tree/main repo, bot it contains docs for 2.X version.

In order to simplify our work with docs for SDK v4, I propose to use `/docs` directory of this repository as more accessible for developers who contribute to SDK repository. On release we can optionally extract all docs to the separate repository or leave them as is. WDYT?

**Comment by irazasyed** [comment]
Please standby, I'm in the process of setup a new docs site powered by [Docusaurus](https://docusaurus.io/).

We'll have `2.x` and `4.x` both docs there in one repo. Might archive the `docs` repo.
